### PR TITLE
add a quirk for king of fans zigbee receiver

### PR DESCRIPTION
**Pull request by igorbernstein2**
The King of Fans ceiling fan receiver does not generate default replies. This module overrides all server commands that do not have a mandatory reply to not expect replies at all.

### CODE REVIEW

File: zigpy/quirks/kof/__init__.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    A simple mixin that allows a cluster to have configureable list of command
+    ids that do not generate an explicit reply.
+    """
+    void_input_commands = []
+
+    def command(self, command, *args, manufacturer=None, expect_reply=None):
+        """
+        Overrides Cluster#command to configure expect_reply behavior based on
+        void_input_commands. Note that this method changes the default value of
+        expect_reply to None. This allows allows the caller to explicitly force
```
Comment (AndreasBomholtz) [inline zigpy/quirks/kof/__init__.py]: Double 'allows'

File: zigpy/quirks/kof/__init__.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        expect_reply to true.
+        """
+        if expect_reply is not None:
```
Comment (Adminiuga) [inline zigpy/quirks/kof/__init__.py]: can we do `if expect_reply is None` and move else block into then block?

**Comment by cgarwood** [comment]
Any updates on this? I just set up one of these receivers and am getting a bunch of timeout errors in the log, I'm hopeful that this will take care of it.

**Comment by Adminiuga** [comment]
running it in my environment, no complains so far.

**Comment by igorbernstein2** [comment]
Finally found time to finish this. All feedback should be addressed

**Comment by igorbernstein2** [comment]
Friendly ping

**Pull request merged**